proprietary/react-icons/CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+<!-- @license CC0-1.0 -->
+
+# Contributing Guidelines
+
+## Contents
+
+- [Exporting icons](#exporting-icons)
+
+## Exporting icons
+
+To export all icons from [the Figma file](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=6852-5124) to SVG use the native SVG export functionality in Figma. To select all icons select the shapes frame, press `Enter` to select direct children and then `Enter` again to select all icon components. The select export to SVG in the sidebar and export all icons to the `icons` folder. Then run the following command to optimize the SVG files:
```

Comment:
```suggestion
## Importing icons from Figma
We create and update icons in Figma, then import them into this package for distribution.
**Important**: Only export icons from Figma for use in this package. Do not use these icons directly in an application.
To export all icons from [the Figma file](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=6852-5124), use Figma's built-in SVG export feature.
Select the Shape frame, press `Enter` to select direct children and then press `Enter` again to select all icon components.
In the right-hand sidebar, click the plus sign next to Export. Choose SVG and export.
Copy these files over to the `icons` folder and run the following command to optimize them:
```

Reply:
This is not correct, I've described how you can export multiple files, you've changed it to selecting the shape which has no children.